internal/testutil/datagen/deployments/scraper.yaml

```diff
@@ -0,0 +1,74 @@
+apiVersion: v1
```

Comment:
TODO: The idea is to put this in the e2e chart, pending @paologallinaharbur thoughs on that.

Reply:
Done